andela/lotus-ah-frontend
# chore(set up test functionality): Setup test functionality with jest …

#### What does this PR do?
 - setup test functionality with jest and enzyme
 -  Write a dummy example test

#### Description of Task to be completed?
- add Jest configuration
- add enzyme and react dependencies
- setup a dummy test in the  ```__tests__``` folder

#### How should this be manually tested?
- Install dependencies ```yarn add```
- start the test by running ```yarn test``` or ```yarn test:watch```
the test should return and show you the result in the terminal

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[#160451550](https://www.pivotaltracker.com/story/show/160451550, "setup test functionality")

#### Screenshots (if appropriate)


#### Questions:
